gk/main.c

```diff
@@ -1483,232 +1776,92 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 			continue;
 		}
 
-		ip_flow_hash_val = likely(front->rss) ? pkt->hash.rss :
-			rss_ip_flow_hf(&packet.flow, 0, 0);
+		pkt_arr[num_ip_flows] = packet;
```

Comment:
Instead of copying `packet`, just make `packet` a pointer and assign `&pkt_arr[num_ip_flows]` to it.